internal/fs/inode/file.go

```diff
@@ -380,7 +385,9 @@ func (f *FileInode) Attributes(
 	attrs.Atime = attrs.Mtime
 	attrs.Ctime = attrs.Mtime
 
-	// Avoid clobbered check and return the attributes for local files.
+	// Clobbered check makes a stat call to GCS. Avoiding stat calls to GCS for
```

Comment:
Should we avoid this stat call incase of local file inode in Sync menthod (line 558) as well?

Reply:
Before syncing file to GCS, we check if the object is still valid (i.e, generation/metageneration). In case of local files, we will check if the object is still not present on GCS(not created via any other flow). Hence stat call is required.